lib/record_loader/primer_panel_loader.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# This file was automatically generated via `rails g record_loader`
+
+# RecordLoader handles automatic population and updating of database records
+# across different environments
+# @see https://rubydoc.info/github/sanger/record_loader/
+module RecordLoader
+  # Creates the specified plate types if they are not present
```

Comment:
Copy paste error with comment above?
Actually looks like all the existing equivalent files reference 'plate types' in this comment (barcode_printer_type_loader, request_type_loader etc.).

Reply:
Oooh... that's probably actually a record loader templating bug. I don't think I copy pasted anything here.